<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flasher</class>
 <widget class="QMainWindow" name="Flasher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flasher</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="usernameEdit">
      <property name="placeholderText">
       <string>Enter your raise.dev username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Enter your raise.dev password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="firmwareUrlEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Firmware URL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="downloadButton">
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="serialPortBox">
      <property name="placeholderText">
       <string>Select serial port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="monitorButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Monitor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="flashButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Flash</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="outputEdit">
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Output</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>333</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Flasher</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuPreferences"/>
  </widget>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
